src/platform/forms-system/src/js/review/ReviewChapters.jsx

```diff
@@ -64,6 +67,15 @@ class ReviewChapters extends React.Component {
     }
   };
 
+  checkValidation = () => {
+    const { form, pageList } = this.props;
+    const { errors } = isValidForm(form, pageList);
+    this.props.setFormErrors({
+      rawErrors: errors,
+      errors: reduceErrors(errors, pageList),
+    });
+  };
```

Comment:
In this case I'm in favor of using redux if it means we can avoid a series of props that exist _only_ to pass something from a grandparent to a child component. Keeping things more tightly contained like that should make the code more maintainable when the component and its props change in the future.

Reply:
If I recall, I did this originally because `ReviewChapters` is connected to the Redux store while `ReviewCollabsilbleChapter` is not. I could have connected the later component to Redux, but then adding a store to the unit test would have been a bigger chore than I wanted to tackle. So I opted for the easier (lazy) method.

I'll work on moving it.